### ISSUE TITLE
QUESTION: Is language definition mandatory in ADM? - What if only music?

### ISSUE THREAD

**Issue by WernerBleisteiner** [kickermeister/wav2bw64#8]
When creating a new ADM with the tool, the default language is English. 
However, when you want to create a music-only ADM, any language selection seems to inappropriate
- disregarding the fact, that music ist regarded "an international language" or "a language of its own ;)

**Comment by kickermeister** [comment]
Good point. Language is optional, so I think I will add a "None" selection option to the dropdown which would cover your use case.
